inc/Engine/Optimization/RUCSS/Admin/Settings.php

```diff
@@ -518,11 +556,11 @@ public function display_no_table_notice() {
 		}
 
 		// translators: %2$s = table name, %3$s = support url.
-		$main_message = __( 'Could not create the %2$s table in the database which is necessary for the Remove Unused CSS feature to work. Please reach out to <a href="%3$s">our support</a>.', 'rocket' );
+		$main_message = __( '%1$s: Could not create the %2$s table in the database which is necessary for the Remove Unused CSS feature to work. Please reach out to <a href="%3$s">our support</a>.', 'rocket' );
```

Comment:
Update the above translators comment to have %1$s

Reply:
Also I think it'd be better to remove any html from the message and use `esc_html__` to be something like that:
```suggestion
		$main_message = esc_html__( '%1$s: Could not create the %2$s table in the database which is necessary for the Remove Unused CSS feature to work. Please reach out to %3$sour support%4$s.', 'rocket' );
```
so %3$s => opening tag with the href (make target="_blank" and rel="noopener")
and %4$s => closing tag

It's a security enhancement, what do u think?